MIGRATION.md

```diff
@@ -74,8 +75,73 @@
     - [Packages renaming](#packages-renaming)
     - [Deprecated embedded addons](#deprecated-embedded-addons)
 
+
 ## From version 5.2.x to 5.3.x
 
+### To tri-config
+
+In storybook 5.3 3 new files for configuration were introduced, that replaced some previous files.
+
+These files are now soft-deprecated, (*they still work, but over time we will promote users to migrate*):
+
+- `config.js` has been renamed to `preview.js`.
+- `addons.js` has been renamed to `manager.js`.
+- `presets.js` has been renamed to `main.js`.
+
+#### Using main.js
+
+this is what a basic `main.js` looks like:
```

Comment:
Now `main.js` is now the main point of configuration for Storybook. This [...]

Reply:
✅